aptos-move/framework/aptos-stdlib/sources/any.move

```diff
@@ -43,6 +44,28 @@ module aptos_std::any {
         &x.type_name
     }
 
+    /// Deserialize primitive types
```

Comment:
I'm sorry but we should not have this API in `Any`.

Please also check the discussion w/ @movekevin and @vgao1996 on the channel, but the principle of this data type is that the only way you can create an instance is via `pack<T>` which ensures type name and serialized bytes align. This makes this entire construct type safe.

Restricting this to primitives makes it a little less unsafe, but still this API now allows arbitrary conversions between primitive types.

I think the case you are using it for in PropertyMap is anyway marginal, since `PropertyValue` is already passed in and out. What you could do, alternatively, make the entire `PropertyValue` based on `Any`, and then offer typed APIs:

```
struct PropertyValue { value: Any }
struct PropertyMap

fun add_u64(map: &mut PropertyMap, key: String, x: u64) {
   let val = PropertyValue { value: any::pack(x) }; internal_add(map, key, val);
}
fun add_u8
fun add_address
fun add_string

fun get_u64(map: &PropertyMap, key: String): u64 {
  any::unpack<u64>(internal_get(map, key);
}
fun get_u8 
...
```

This will potentially create an explosion in the number of user txns but it is the cleanest IMO.

Reply:
The problem here is that users will pass a vector<u8> as property value. we don't have a way to call `add_u64()` to store the value. 

If any is not the right place to add the primitive type deserializer. I could create a new lib for deserializing primitive types to prevent the misuse of Any lib.